pytket/qir/conversion/api.py

```diff
@@ -37,29 +37,27 @@ class ReturnTypeQIR(Enum):
 def pytket_to_qir(
     circ: Circuit,
     name: str = "Generated from input pytket circuit",
-    returntype: ReturnTypeQIR = ReturnTypeQIR.BINARY,
+    returntype: QIRFormat = QIRFormat.BINARY,
```

Comment:
How about renaming `returntype` to `format`?

Reply:
done in @cqc-melf
[rename format parameter](https://github.com/CQCL/pytket-qir/pull/36/commits/2357a71e06280cf9b1dbaff53fb2acfd393f0866)
[2357a71](https://github.com/CQCL/pytket-qir/pull/36/commits/2357a71e06280cf9b1dbaff53fb2acfd393f0866)

Reply:
pylint does not like that, would do qir_format instead? (done in @cqc-melf
[update format parameter name](https://github.com/CQCL/pytket-qir/pull/36/commits/a1e7c6e15651d02258f6607a31c7b45f70a7f293)
[a1e7c6e](https://github.com/CQCL/pytket-qir/pull/36/commits/a1e7c6e15651d02258f6607a31c7b45f70a7f293) )